docs/src/ReactPlayground.js

```diff
@@ -6,6 +6,8 @@ const classNames = require('classnames');
 const React = require('react');
 const ReactDOM = require('react-dom');
 
+const bootstrapUtils = require('../../src/utils/bootstrapUtils.js').default;
```

Comment:
How should we handle this? Everything on a default export, or a bunch of named exports? i.e. when using ES module shims, should it be

```js
import { * as bootstrapUtils } from './utils/bootstrapUtils';
```

or

```js
import bootstrapUtils from './utils/bootstrapUtils';
```